main.py

```diff
@@ -95,6 +95,30 @@ class SymbolKind(object):
 }
 
 
+class CompletionItemKind(object):
+    Text = 1
+    Method = 2
+    Function = 3
+    Constructor = 4
+    Field = 5
+    Variable = 6
+    Class = 7
+    Interface = 8
+    Module = 9
+    Property = 10
+    Unit = 11
+    Value = 12
+    Enum = 13
+    Keyword = 14
+    Snippet = 15
+    Color = 16
+    File = 17
+    Reference = 18
+
+
+completion_item_kind_names = {v: k for k, v in CompletionItemKind.__dict__.items()}
```

Comment:
Why not just write it as a dict in the first place?

Reply:
or [enum](https://docs.python.org/3.4/library/enum.html).

Reply:
Had wondered about Enum before, but documentation said "new in version 3.4".
Tried it now, getting an ImportError:
```
  File "/Users/tomv/Library/Application Support/Sublime Text 3/Packages/LSP/main.py", line 7, in <module>
    from enum import Enum
ImportError: No module named 'enum'
```

Reply:
Ai...forget that 3.3 doesn't support enum.

Reply:
It was easiest to paste and fix the list from the spec or typescript and then add the one-liner to create the reverse lookup. 
I agree the lookup is the only thing we use right now, but I'd like an enum-like lookup for future functionality.

Reply:
Fair enough 👍